### PR TITLE
Unable to duplicate a serving runtime

### PR DESCRIPTION
**Pull request by dpanshug**
Closes: [RHOAIENG-552](https://issues.redhat.com/browse/RHOAIENG-552)

## Description
Changed the label for Duplicating serving runtime

image

## How Has This Been Tested?
1. Create an OOTB serving runtime
2. Try duplicating it

## Test Impact
Added integration test to check page headings for Add, Duplicate and Edit.

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [x] Included any necessary screenshots or gifs if it was a UI change.
- [x] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

@vconzola

### CODE REVIEW

File: frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimeAddTemplate.tsx [hunk 1]
```diff
@@ -211,7 +213,7 @@ const CustomServingRuntimeAddTemplate: React.FC<CustomServingRuntimeAddTemplateP
                     });
                 }}
               >
-                {existingTemplate ? 'Update' : 'Add'}
+                {existingTemplate ? 'Update' : state ? 'Duplicate' : 'Add'}
```
Comment (lucferbux) [inline frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimeAddTemplate.tsx:216]: I think @vconzola commented to change "Add" to create.

**Review by lucferbux — CHANGES_REQUESTED** [review]
Changes look great, we need testing coverage and we should tag @vconzola to review the changes.

**Comment by lucferbux** [comment]
This is not only a label change, que should add coverage to tests that adding a new SR will display "Add" and if we try to duplicate it will get "Duplicate..."

**Comment by vconzola** [comment]
The page title change looks OK, but you should also change the description text below the title. The text should say, "Add a new, editable runtime by duplicating and existing runtime."

**Comment by dpanshug** [comment]
@lucferbux, Cypess test added and also refactored the test file according to [cypress test structure](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Test-Structure)

**Review by lucferbux — APPROVED** [review]
/lgtm

Great job here!

**Pull request merged**